docs/backend-api/api-methods.md

```diff
@@ -0,0 +1,211 @@
+# API Methods
+All API methods calls except for token generation are secured via [OAuth2](/docs/backend-api/api-authentication-oauth2.md).
+You need to provide token via `Authorization` header to access requested data like this: ```Authorization: Bearer eyJ0eXAiOiJKV...s3SKg```
+
+## List of methods
+- [Retrive access token](#retrieve-access-token)
+- [Retrieve a product](#retrieve-a-product)
+- [Retrieve a list of products](#retrieve-a-list-of-products)
+
+**Retrieve access token**
+----
+
+* **URL**
+
+  `/api/token`
+
+* **Method:**
+
+  `POST`
+
+*  **Data Params**
+
+   **Required:**
+
+   `grant_type=client_credentials`
+   `client_id=[string]`
+   `client_secret=[string]`
+
+* **Success Response:**
+
+  * **Code:** `200` <br />
+     **Content:**
+        ```
+        {
+            "token_type": "Bearer",
+            "expires_in": 3600,
+            "access_token": "eyJ0eXAiOiJKV...s3SKg"
+        }
+        ```
+
+* **Error Response:**
+
+  * **Code:** `401 UNAUTHORIZED` <br />
+    **Content:**
+    ```
+    {
+        "error": "invalid_client",
+        "message": "Client authentication failed"
+    }
+    ```
+
+  OR
+
+  * **Code:** `400 Bad Request` <br />
+
+* **Notes:**
+Token is valid for one hour.
+You will need to generate new token after the first one expires.
+
+**Retrieve a product**
+----
+
+* **URL**
+
+  `/api/v1/products/{uuid}`
+
+* **Method:**
+
+  `GET`
+
+*  **URL Params**
+
+   **Required:**
+
+   `uuid=[string]`
+
+* **Success Response:**
+
+  * **Code:** `200` <br />
+     **Content:**
```

Comment:
Why is the content part not structured and in the article it is one very long text?

Reply:
mistake in indentation, fixed